include/miopen/miopen.h

```diff
@@ -103,6 +103,7 @@ typedef enum {
     miopenStatusNotImplemented = 6, /*!< Use of unimplemented feature. */
     miopenStatusUnknownError   = 7, /*!< Unknown error occurred. */
     miopenStatusUnsupportedOp  = 8, /*!< Unsupported operator for fusion. */
+    miopenStatusNoGPU          = 9, /*!< GPU operations skipped. */
```

Comment:
```suggestion
    miopenStatusGpuOperationsSkipped          = 9, /*!< This is not an error. */
```

In fact, I doubt it. The library did exactly what was requested. It can complete the `Find()` call normally and simply return an empty list of found solutions and a success code. But let's keep dedicated code as we just discussed.